src/pages/index.astro

```diff
@@ -82,14 +88,14 @@ if (session && session.session.full) {
 						<article>
 							<img src='/icons/sound.png' alt='Sound icon' />
 							<div>
-								<h3>Sound &ndash MAX98357A</h3>
+								<h3>Sound - MAX98357A</h3>
 								<p>5 channels, 4 waveforms</p>
 							</div>
 						</article>
 						<article>
 							<img src='/icons/display.png' alt='Display icon' />
 							<div>
-								<h3>Display &ndash TFT7735</h3>
+								<h3>Display - TFT7735</h3>
 								<p>160&times128 color LCD</p>
 							</div>
 						</article>
```

Comment:
I think it's better to have the HTML entity, more clear in IDEs. (Same in line 112 but GitHub wouldn't let me highlight that too.)

Reply:
I agree, the only problem is that those elements currently look like so:
![image](https://user-images.githubusercontent.com/70044734/229003002-1336f2ac-0dbd-4db5-b11d-e2b6d95ede4e.png)